settings.py

```diff
@@ -303,7 +299,8 @@
 SEQR_PRIVACY_VERSION = float(os.environ.get('SEQR_PRIVACY_VERSION', 1.1))
 SEQR_TOS_VERSION = float(os.environ.get('SEQR_TOS_VERSION', 1.2))
 
-BASE_URL = os.environ.get("BASE_URL", "/")
+BASE_URL = os.environ.get("BASE_URL", "http://localhost/")
```

Comment:
rather than changing this, especially since we didn;t have a CSRF issue locally, it might be better to just put the `CSRF_TRUSTED_ORIGINS` under an if statement for the presence of the base url

Reply:
I put it under the same `if DEPLOYMENT_TYPE in {'prod', 'dev'}:` check we do earlier in the file. I could also check if BASE_URL != '/'.